ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/QueryFilterNode.java

```diff
@@ -47,6 +49,7 @@
 import io.confluent.ksql.schema.ksql.DefaultSqlValueCoercer;
 import io.confluent.ksql.schema.ksql.LogicalSchema;
 import io.confluent.ksql.schema.ksql.SystemColumns;
+import io.confluent.ksql.schema.ksql.types.SqlBaseType;
```

Comment:
The RQTT tests are definitely good, though it might make sense to add a test or two to the unit tests for `QueryFilterNodeTest`.

Reply:
Added some unit tests.